imap_processing/cdf/config/imap_hi_variable_attrs.yaml

```diff
@@ -312,6 +312,20 @@ hi_pset_esa_energy_step:
   VAR_TYPE: support_data
   dtype: uint8
 
+hi_pset_calibration_prod:
+  CATDESC: Calibration product number
+  FIELDNAM: Coincidence types are combined into various calibration products
+  FILLVAL: 65535
+  FORMAT: I1
+  LABLAXIS: Calibration Prod
+  UNITS: " "
+  SCALE_TYP: linear
+  MONOTON: INCREASE
+  VALIDMIN: 0
+  VALIDMAX: 4
```

Comment:
Could the max possibly change and if so does it make sense to have this be a larger value?